mysqlfragfinder.sh

```diff
@@ -121,7 +140,9 @@ for i in ${databases[@]}; do
 			for table in ${fragmented[@]}; do
 				let fraggedTables=$fraggedTables+1;
 				echo -ne "\tOptimizing $table ... ";
-				"$mysqlCmd" -u"$mysqlUser" -p"$mysqlPass" -h"$mysqlHost" -D "$i" --skip-column-names --batch -e "optimize table \`$table\`" 2>"$log" >/dev/null
+
+				$mysqlCmd -D "$i" --skip-column-names --batch -e "optimize table \`$table\`" 2>"$log" >/dev/null
```

Comment:
'2>"$log" >/dev/null' sends only stderr to the logfile and not stdout, so half of the conversation is missing.